_includes/2017_data/sponsors.html

```diff
@@ -1,45 +1,65 @@
-i<!-- Clients Aside -->
+<!-- Clients Aside -->
 
     <aside class="clients" class="bg-light-gray">
+
         <div class="container">
+
             <div class="row">
                 <div class="col-lg-12 text-center">
                     <h2 class="section-heading" id="sponsors">Sponsors</h2>
                 </div>
             </div>
+
             <div class="row">
                 <div class="col-md-12 text-center">
                     <h3 class="section-heading">Metropolis-Hastings Tier</h3>
                 </div>
             </div>
 
-            <div class = "row">
-	    	<div class = "col-md-4 col-sm-6">
-			<a href = "https://zulip.com/" class = "portfolio-link" data-toggle = "modal">
-			<img src = "./css/2017_style/img/sponsors/zulip.png" class = "image-responsive img-centered" alt = "zulip">
-			</a>
-		</div>
-	    </div>
             <div class="row">
-                <div class="col-md-4 col-sm-6">
-                </div>
                 <div class="col-md-4 col-sm-6">
                     <a href="https://tidelift.com/" class="portfolio-link" data-toggle="modal">
                         <img src="./css/2017_style/img/sponsors/Tidelift_primary-logo.png" class="img-responsive img-centered" alt="tidelift">
                     </a>
                 </div>
-                <!--
+		    </div>
+
+            <div class = "row">
+                <div class = "col-lg-12 text-center">
+                    <h2 class = "section-heading" id = "techpartner">Technology Partners</h2>
+                </div>
+            </div>
+
+            <div class = "row">
+                <div class = "col-md-12 text-center">
+                    <h3 class = "section-heading">Zulip</h3>
+                </div>
+            </div>
```

Comment:
I think it's not necessary to add a header for each company, the logo is enough

Reply:
Ok, so should I remove the headings then?

Reply:
not the headings, only "zulip" as heading, the tier heading are already right